styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
+  border: 0;
+  border-top: 1px solid $gray-lighter;
+  margin-bottom: $line-height-computed / 4;
+  margin-top:    $line-height-computed / 4;
```

Comment:
I still don't think that's a good idea. I'd like to maybe have a larger group conversation about it? Can we schedule a wheaties conversation around it?